app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+import os
+
+
+def make_directories(destination: str) -> None:
+    directories = destination.split("/")[:-1]
+    path = ""
+    for directory in directories:
+        path += directory + "/"
+        if not os.path.exists(path):
+            os.mkdir(path)
```

Comment:
You can specify this code.
```suggestion
    path = os.path.join(directories)
    os. makedirs(path, exist_ok=True)
```